### PR TITLE
Adrian/reader

### PR DESCRIPTION
**Pull request by adrianovi**

### CODE REVIEW

File: src/main/java/org/fundacionjala/at15/pokemon/io/Reader.java [hunk 1]
```diff
@@ -7,12 +7,18 @@
...
             File path = PathHandler.getPath(fileName);
-            String line = PathHandler.jSonStringReader(path);
-            return ReaderJSON.readJSON(subName, line);
+            String text = PathHandler.stringReader(path);
+            return ReaderJSON.readJSON(subName, text);
+        }
+        if (PathHandler.getTypeFile(fileName).equals("xml")) {
+            File path = PathHandler.getPathXML(fileName);
+            String text = PathHandler.stringReader(path);
+            return ReaderJSON.readJSON(subName, text);
```
Comment (adhes14) [inline src/main/java/org/fundacionjala/at15/pokemon/io/Reader.java]: Probably here you must return ReaderXML

**Review by adhes14 — CHANGES_REQUESTED** [review]
Please see my comment

File: src/main/java/org/fundacionjala/at15/pokemon/io/Reader.java (hunk 1 above)
Comment (adrianovi) [reply src/main/java/org/fundacionjala/at15/pokemon/io/Reader.java]: That's right, I'll change it

**Review by adhes14 — APPROVED** [review]
Great work!

**Pull request merged**